persistence-modules/spring-data-jpa/src/test/java/com/baeldung/dao/repositories/UserRepositoryIntegrationTest.java

```diff
@@ -390,6 +391,28 @@ public void givenUsersInDBWhenFindByEmailsWithDynamicQueryThenReturnCollection()
 
         assertThat(usersWithEmails.size()).isEqualTo(2);
     }
+    
+    @Test
+    public void givenUsersInDBWhenFindByNameListReturnCollection() {
+
+        User user1 = new User();
+        user1.setName(USER_NAME_ADAM);
+        user1.setEmail(USER_EMAIL);
+        userRepository.save(user1);
+
+        User user2 = new User();
+        user2.setName(USER_NAME_PETER);
+        user2.setEmail(USER_EMAIL2);
+        userRepository.save(user2);
+
+        List<String> names = new ArrayList<>();
+        names.add(USER_NAME_ADAM);
+        names.add(USER_NAME_PETER);
```

Comment:
`MINOR` - consider whether it might be cleaner to go:

`List<String> names = Arrays.asList(USER_NAME_ADAM, USER_NAME_PETER);`